Reference/conformance/conformance.pb.swift

```diff
@@ -140,6 +140,9 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message {
 
   init() {}
 
+  /// Used by the SwiftProtobuf decoding machinery, not generally used directly.
+  /// `init(serializedData:)`, `init(jsonUTF8Data:)`, and other decoding initializers
+  /// are defined in `Message` and `Message+*Additions` files.
```

Comment:
I think this comment needs to specifically mention the "SwiftProtobuf library" somewhere.  Perhaps

> Used by the decoding initializers in the SwiftProtobuf library. ...

Or perhaps:

> ... are defined in the SwiftProtobuf library.  See the `Message` and Message+*Additions` files.